packages/services/src/competition.service.ts

```diff
@@ -1222,7 +1186,9 @@ export class CompetitionService {
         imageUrl: agent.imageUrl,
         score,
         active: isActive,
-        deactivationReason: !isActive ? agent.deactivationReason : null,
+        deactivationReason: !isActive
+          ? (competitionDeactivationReason ?? null)
```

Comment:
the `(competitionDeactivationReason ?? null)` seems redundant since it falls back to `null`?